features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/modules/operation.js

```diff
@@ -27,7 +27,42 @@ var operationModule = function () {
     var publicMethods = {};
     var privateMethods = {};
 
+    privateMethods.getOperationsFromFeatures = function (deviceType) {
+        var GenericFeatureManager = Packages.org.wso2.carbon.apimgt.webapp.publisher.feature.management.GenericFeatureManager;
+        try {
+            var featureManager = GenericFeatureManager.getInstance();
+            var features = featureManager.getFeatures(deviceType);
+            var featureList = [];
+            var feature;
+            for (var i = 0; i < features.size(); i++) {
+                feature = {};
+                feature["id"] = features.get(i).getId();
+                feature["code"] = features.get(i).getCode();
+                feature["name"] = features.get(i).getName();
+                feature["description"] = features.get(i).getDescription();
+                feature["deviceType"] = features.get(i).getDeviceType();
+                feature["metadataEntries"] = [];
+                var metaData = features.get(i).getMetadataEntries();
+                if (metaData && metaData != null) {
```

Comment:
Fixed in commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1